internal/pkg/cli/task_run.go

```diff
@@ -124,7 +124,7 @@ func (o *runTaskOpts) Validate() error {
 
 // Ask prompts the user for any required or important fields that are not provided.
 func (o *runTaskOpts) Ask() error {
-	if err := o.askTaskFamilyName(); err != nil {
+	if err := o.askTaskGroupName(); err != nil {
 		return err
 	}
 	if err := o.askEnvName(); err != nil {
```

Comment:
Can we add a unit test to mock this failure and ensure that the error is wrapped?

Reply:
Also on this method call, can we add a unit test which returns "None" to ensure that everything works as expected in the case that we have to check the default cluster?